### ISSUE TITLE
Inconsistent `add_library` indentation

### ISSUE THREAD

**Issue by mguzzina** [BlankSpruce/gersemi#17]
I noticed that the `add_library` function gets formatted slightly differently depending on the labels:

```cmake
add_library(
    foo
    INTERFACE
        include/File1.hpp
        include/File2.hpp
        include/File3.hpp
        include/File4.hpp
        include/File5.hpp
)
```

With many other labels this becomes something like

```cmake
add_library(
    foo
    SHARED
    src/File1.cpp
    src/File2.cpp
    src/File3.cpp
    src/File4.cpp
    src/File5.cpp
)
```

I've tried with `SHARED`, `STATIC`, `OBJECT` and `INTERFACE`. `INTERFACE` seems the only one that is "correctly" formatted.

Is this intended?

**Comment by BlankSpruce** [comment]
Definitely not. I'll take a look at that later this week.

**Comment by BlankSpruce** [comment]
Probably not intended although I'd guess that SHARED/STATIC/OBJECT have the correct one in this case. I'll take a look at that later this week.

**Comment by BlankSpruce** [comment]
As I've thought `INTERFACE` case had wrong formatting. In contrast to command like `target_link_libraries` this keyword doesn't introduce group of arguments with narrow semantics. Let's consider this following code and what it would look like in imaginary Python based build system generator:
```cmake
target_link_libraries(
    FOO
    INTERFACE
        bar
        baz
)

add_library(
    FOO
    INTERFACE
    bar
    baz
)
```
In this imaginary generator this would be equivalent to:
```python
target_link_libraries(
    name="FOO", 
    interface_links=["bar", "baz"], 
    public_links=[], # explicit arguments to show the idea
    private_links=[]
)

add_library(
     name="FOO", 
     kind="INTERFACE", 
     sources=["bar", "baz"]
)
```

This will be fixed in 0.12.0 but the release will be slightly postponed due to new feature related to #15.
